incubator/java-openliberty/templates/default/src/main/liberty/config/server.xml

```diff
@@ -1,6 +1,7 @@
 <server description="Liberty server">
     <featureManager>
-        <feature>microProfile-3.2</feature>
+        <feature>mpHealth-2.0</feature>
+        <feature>mpMetrics-2.0</feature>
```

Comment:
Two thoughts:

1) This should probably have at least jaxrs-2.1 to enable the JAX-RS resource in the template (which should probably have an IT, though that could be a separate issue for whenever)

2) Though this is just a template it's got the "breaking" characteristic that it removing certain API/features.    Also there's not any particular reason it needs to be merged together with the base stack change.   Maybe then it'd be better to save such a change for a separate PR, minor update.      This way our v0.1.1 change can be described identically for new & existing projects.    (Though this won't always be possible, so not 100% sure what I'm proposing is the right thing. What do you think?)

Reply:
I think it would be best to leave microProfile-3.2 as the feature definition here. In a separate PR I can slim this down to mpHealth, mpMetrics, and jaxrs while adding an IT to test the app endpoint.

Reply:
I re-added the microProfile-3.2 feature and opened https://github.com/appsody/stacks/issues/592 to slim down the feature list and add an IT to test jaxrs.